pkg/solana/txm/txm.go

```diff
@@ -602,19 +692,20 @@ func (txm *Txm) Enqueue(ctx context.Context, accountID string, tx *solanaGo.Tran
 		}
 	}
 
-	// Use transaction ID provided by caller if set
-	id := uuid.New().String()
-	if txID != nil && *txID != "" {
-		id = *txID
-	}
-	msg := pendingTx{
+	msg := &pendingTx{
```

Comment:
Is there a benefit to using a pointer to `pendingTx` here if we're just dereferencing it before sending it over the channel?

Reply:
thanks for this!